Github Issue Reader/ViewController.swift

```diff
@@ -123,18 +124,35 @@ class ViewController: UIViewController {
             return
         }
         
-        viewModel.fetchIssues(for: organization, repo: repo) { [self] issues in
-            print("Successfully received issues.")
+        activityIndicator.startAnimating()
+        DispatchQueue.main.asyncAfter(deadline: .now() + 5) {
             
-            let layout = UICollectionViewFlowLayout()
-            
-            let issuesVC = IssuesCollectionVC(collectionViewLayout: layout)
-            
-            issuesVC.viewModel = viewModel
-            print("Presenting ViewController")
-            navigationController?.pushViewController(issuesVC, animated: true)
+            self.viewModel.fetchIssues(for: organization, repo: repo) { [self] issues in
+                print("Successfully received issues.")
+                
+                let layout = UICollectionViewFlowLayout()
+                
+                let issuesVC = IssuesCollectionVC(collectionViewLayout: layout)
+                
+                issuesVC.viewModel = viewModel
+                print("Presenting ViewController")
+                navigationController?.pushViewController(issuesVC, animated: true)
+                activityIndicator.stopAnimating()
+            }
         }
     }
     
+    let activityIndicator = UIActivityIndicatorView(style: .large)
+    
+    func activateIndicator() {
```

Comment:
"activate" is probably overstating what happens here. Perhaps "setup" or "configure"